requirements.txt

```diff
@@ -1,16 +1,14 @@
 # Requirements for automatic testing and building
 # Runtime requirements are described in the spec file
 
-pip
-setuptools
 coverage
 coveralls
-ordered-set
 polib
 pylint
```

Comment:
You can put the test dependencies as `test_requires` in `setup.py`.

Reply:
The `test` command was deprecated in `setuptools` v41.5.0. Which means, so has `test_requires`. See https://setuptools.readthedocs.io/en/latest/setuptools.html

I suggest using `extra requires` and creating a group called "test".